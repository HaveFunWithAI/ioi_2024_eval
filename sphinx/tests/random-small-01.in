50 49
40 5 16 40 16 16 16 40 5 16 16 16 5 40 40 16 16 16 40 16 5 5 5 5 5 5 5 5 5 40 5 16 40 40 16 5 16 40 40 5 5 16 5 40 5 16 40 5 5 40
27 40
29 40
40 43
24 29
40 45
25 29
25 49
16 25
42 43
40 41
24 26
33 42
2 42
0 25
20 24
8 33
20 38
47 49
34 43
8 17
6 20
22 34
0 19
39 49
20 37
23 45
20 32
12 42
7 29
3 49
20 21
3 31
9 39
36 43
4 36
15 40
30 33
11 23
14 21
1 17
19 44
10 30
26 35
13 32
5 40
0 46
28 41
14 48
18 31
