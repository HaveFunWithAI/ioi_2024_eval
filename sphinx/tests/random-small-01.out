50 375
40 5 16 40 16 16 16 40 5 16 16 16 5 40 40 16 16 16 40 16 5 5 5 5 5 5 5 5 5 40 5 16 40 40 16 5 16 40 40 5 5 16 5 40 5 16 40 5 5 40
